package/gluon-mesh-batman-adv-core/luasrc/lib/gluon/upgrade/310-gluon-mesh-batman-adv-core-mesh

```diff
@@ -51,9 +51,22 @@ uci:section('network', 'interface', 'bat0',
 		    proto = 'none',
 		    macaddr = sysconfig.primary_mac,
 		    multicast_router = 2,
+		    learning = 0,
 	    }
 )
 
+uci:delete('network', 'client_lan')
+if sysconfig.lan_ifname then
+	uci:section('network', 'interface', 'client_lan',
+		    {
+			    unicast_flood = 0,
+		    }
+	)
+	for _, lanif in ipairs(lutil.split(sysconfig.lan_ifname, ' ')) do
+		uci:set('network', 'client_lan', 'ifname', lanif)
```

Comment:
1. We should investigate why the list doesn't work here
2. In any case, this code is plain wrong. If you want to use an option, use `uci:set('network', 'client_lan', 'ifname', sysconfig.lan_ifname)`, your loop will replace the old value in each run otherwise!